examples/tutorials/analysis-time/Variability_estimation.py

```diff
@@ -0,0 +1,193 @@
+    * np.sqrt(
+        (f_max_err / f_mean) ** 2
+        + (f_min_err / f_mean) ** 2
+        + ((np.std(flux, ddof=1) / np.sqrt(len(flux))) / (f_max - f_mean)) ** 2
+        * (variability_amplitude_100 / 100) ** 4
+    )
+)
+
+variability_amplitude_significance = (
+    variability_amplitude_100 / variability_amplitude_error
+)
+
+print(variability_amplitude_significance)
+
+######################################################################
+#  Fractional excess variance, point-to-point fractional variance and doubling/halving time
+# ----------------------------------------------
+# The fractional excess variance ([reference
+# paper](https://ui.adsabs.harvard.edu/abs/2003MNRAS.345.1271V/abstract)) is a simple but effective method to assess
+# the significance of a time variability feature in an object, for example an AGN flare.
+# It is important to note that it requires gaussian errors to be applicable.
+# The excess variance computation is implemented in `~gammapy.estimators.utils`.
+# A similar estimator is the point-to-point fractional variance, which samples the lightcurve with smaller time granularity.
+# In general, the point-to-point fractional variance being higher than the fractional excess variance is indicative of the presence of very short timescale variability.
+#
+# The doubling and halving time of the light curve can also be computed.
+# This provides information on the shape of the variability feature.
+
+fvar_table = compute_lightcurve_fvar(lc_1d)
+print(fvar_table)
+
+fpp_table = compute_lightcurve_fpp(lc_1d)
+print(fpp_table)
+
+dtime_table = compute_lightcurve_doublingtime(lc_1d, flux_quantity="flux")
+print(dtime_table)
+
+######################################################################
+# Bayesian blocks
+# ----------------------------------------------
+# The presence of temporal variability in a lightcurve can be assessed by using bayesian blocks ([reference
+# paper](https://ui.adsabs.harvard.edu/abs/2013ApJ...764..167S/abstract)). A good and simple-to-use implementation of the algorithm is found in `astropy.stats`([documentation](https://docs.astropy.org/en/stable/api/astropy.stats.bayesian_blocks.html)). This implementation uses gaussian statistics, as opposed to the [first introductory paper](https://iopscience.iop.org/article/10.1086/306064) which was based on poissonian statistics.
+#
+# By passing the flux and error on the flux as `measures` to the method we can obtain the list of optimal bin edges defined by the bayesian blocks algorithm.
+
+time = lc_1d.geom.axes["time"].time_mid.mjd
+
+bayesian_edges = bayesian_blocks(
+    t=time, x=flux.flatten(), sigma=flux_err.flatten(), fitness="measures"
+)
+
+# We can visualize the difference between the original lightcurve and the rebin with bayesian blocks
+
+bayesian_flux = []
+for _ in range(len(bayesian_edges) - 1):
+    mask = (time >= bayesian_edges[_]) & (time <= bayesian_edges[_ + 1])
+    bayesian_flux.append(flux.flatten()[mask].mean())
```

Comment:
How is the mean supposed to be performed?
Should it be a simple mean? Should it be averaged with the respective inverse errors?

Reply:
Rather than using a mask you could also slice:
```
lc_1d.slice_by_time(tmin,tmax).flux.mean()
```

Reply:
A weighted mean would be better, no?

Reply:
Swapped in an inverse weighted average